src/tests/api/DocumentSelectionHappyPath.test.ts

```diff
@@ -15,64 +15,15 @@ describe("E2E Happy Path /documentSelection Endpoint", () => {
-		expect(sessionId).toBeTruthy();
-		const response = await postDocumentSelection(dataPassport, sessionId);
-		expect(response.status).toBe(200);
-		expect(response.data).toBe("Instructions PDF Generated");
-		// Authorization
-		const authResponse = await authorizationGet(sessionId);
-		expect(authResponse.status).toBe(200);
-		expect(authResponse.data.authorizationCode.value).toBeTruthy();
-		expect(authResponse.data.redirect_uri).toBeTruthy();
-		expect(authResponse.data.state).toBeTruthy();
-		// // Post Token
-		const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri );
-		expect(tokenResponse.status).toBe(200);
-		console.log(tokenResponse.data);
-		// Post User Info
-		const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
-		expect(userInfoResponse.status).toBe(202);
-		console.log(userInfoResponse.data);
+		await prepareForCallback(dataPassport, sessionId);
 	});
 
 	it("E2E Happy Path Journey - Drivers Licence", async () => {
-		const response = await postDocumentSelection(dataDriversLicense, sessionId);
-		expect(response.status).toBe(200);
-		expect(response.data).toBe("Instructions PDF Generated");
-		// Authorization
-		const authResponse = await authorizationGet(sessionId);
-		expect(authResponse.status).toBe(200);
-		expect(authResponse.data.authorizationCode.value).toBeTruthy();
-		expect(authResponse.data.redirect_uri).toBeTruthy();
-		expect(authResponse.data.state).toBeTruthy();
-		// // Post Token
-		const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri );
-		expect(tokenResponse.status).toBe(200);
-		console.log(tokenResponse.data);
-		// Post User Info
-		const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
-		expect(userInfoResponse.status).toBe(202);
-		console.log(userInfoResponse.data);
+		await prepareForCallback(dataDriversLicense, sessionId);
 	});
 
 	it("E2E Happy Path Journey - Biometric Residence Permit", async () => {
-		const response = await postDocumentSelection(dataBrp, sessionId);
-		expect(response.status).toBe(200);
-		expect(response.data).toBe("Instructions PDF Generated");
-		// Authorization
-		const authResponse = await authorizationGet(sessionId);
-		expect(authResponse.status).toBe(200);
-		expect(authResponse.data.authorizationCode.value).toBeTruthy();
-		expect(authResponse.data.redirect_uri).toBeTruthy();
-		expect(authResponse.data.state).toBeTruthy();
-		// // Post Token
-		const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri );
-		expect(tokenResponse.status).toBe(200);
-		console.log(tokenResponse.data);
-		// Post User Info
-		const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
-		expect(userInfoResponse.status).toBe(202);
-		console.log(userInfoResponse.data);
+		await prepareForCallback(dataBrp, sessionId);
```

Comment:
Maybe we call it something else if it's being used in here too - naming is not a strength of mine but maybe something like `initiateUserInfo`?

Reply:
Good shout, and initiateUserInfo sounds good to me 👍